addon/components/bs-modal.js

```diff
@@ -267,7 +281,6 @@ export default class Modal extends Component {
    * @public
    */
   @arg
-  transitionDuration = 300;
```

Comment:
I guess you had removed this line by accident?

Reply:
Yes exact!!!